rhizome/host/lib/cloud_hypervisor.rb

```diff
@@ -1,9 +1,10 @@
 # frozen_string_literal: true
 
+require "fileutils"
 require_relative "../../common/lib/arch"
 
 module CloudHypervisor
-  FirmwareClass = Struct.new(:version, :name) {
+  FirmwareClassLegacy = Struct.new(:version, :name) {
```

Comment:
Seems like this the previous commit references this symbol, no? (Not sure how much of this came from my old prototype, which if it did this, can be called a concession to being only a prototype...)

Reply:
oops, my bad. this was supposed to be in the first commit. I thought it already was ... but it isn't ... as you're pointing out.